examples/esp_idf/hello/pytest/README.md

```diff
@@ -0,0 +1,87 @@
+# Hello Test
+
+This is a simple test that determines whether the DUT is able to
+establish a connection with the Golioth backend and send Hello to it.
+The test requires:
+- An attached ESP board with the hello sample flashed on it
```

Comment:
The `board` fixture can flash the device, so it's not required that the hello sample already be flashed.